Mathlib/Data/Int/NatAbs.lean

```diff
@@ -0,0 +1,30 @@
+/-
+Copyright (c) 2021 Anne Baanen. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anne Baanen, Jeremy Tan
+-/
+import Mathlib.Algebra.GroupWithZero.Hom
+import Mathlib.Algebra.GroupWithZero.Nat
+import Mathlib.Algebra.Ring.Int.Defs
+
+/-!
+# Lemmas about `Int.natAbs`
+
+This file contains some results on `Int.natAbs`, the absolute value of an integer as a
+natural number.
+
+## Main results
+
+* `Int.natAbsHom`: `Int.natAbs` bundled as a `MonoidWithZeroHom`.
+-/
+
```

Comment:
I think it would be more idiomatic to have `namespace Int` at the top of the file here, so that the individual defs don't need `Int.` in their names.